api/src/controllers/changes.js

```diff
@@ -164,6 +172,9 @@ const writeDownstream = (feed, content, end) => {
   if (feed.res.finished) {
     return;
   }
+  if (feed.error) {
+    feed.res.status(500);
```

Comment:
Is there a chance we've already sent headers?

Reply:
Good point. Thanks!